packages/app/obojobo-document-engine/src/scripts/oboeditor/components/navigation/more-info-box.scss

```diff
@@ -105,6 +105,10 @@
 					}
 				}
 
+				select {
+					width: 22.7em;
```

Comment:
I think we should go with a flex box solution. This will break as soon as we need to change the dimensions of the more info box.